WBooks/CommentTableViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  CommentTableViewController.swift
+//  WBooks
+//
+//  Created by Nicolas Manograsso on 01/11/2018.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import Networking
+import ReactiveSwift
+import UIKit
+import WolmoCore
+
+class CommentTableViewController: UITableViewController {
+
+    var bookID : Int?
```

Comment:
bookID can never be null when you use it right? you can set the type to `Int!` then